### PR TITLE
Golint cleanup

### PR DESCRIPTION
**Pull request by knarz**
Hey,

I just spent a rather boring train ride reading through the code and doing some cleanup with golint. This first PR is just getting all the low hanging fruit, i.e. fixing variable names and a first effort towards closing #847 .
The bigger task is most likely going to be writing all the comments to make golint happy.

Hope this is not too big of a commit.

### CODE REVIEW

**Comment by sthulb** [comment]
LGTM

File: drivers/openstack/openstack.go [hunk 1]
```diff
@@ -454,8 +454,8 @@ const (
 )
 
 func (d *Driver) checkConfig() error {
-	if d.AuthUrl == "" {
-		return fmt.Errorf(errorMandatoryEnvOrOption, "Authentication URL", "OS_AUTH_URL", "--openstack-auth-url")
+	if d.AuthURL == "" {
+		return fmt.Errorf(errorMandatoryEnvOrOption, "Autentication URL", "OS_AUTH_URL", "--openstack-auth-url")
```
Comment (ehazlett) [inline drivers/openstack/openstack.go]: This should be "Authentication"

**Comment by ehazlett** [comment]
Thanks!  One minor comment -- initial testing looks good.  Thanks a lot!

**Comment by knarz** [comment]
Oops, fixed the spelling error.

**Comment by ehazlett** [comment]
This looks good.  Would you mind squashing these into a single commit? Thanks!

**Comment by ehazlett** [comment]
@knarz why did you close?

**Comment by knarz** [comment]
Because I'm a huge klutz, sorry. Going to open a new one in a second.

**Pull request closed without merging**